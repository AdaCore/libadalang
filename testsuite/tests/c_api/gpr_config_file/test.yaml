driver: c-api
compile_units: [main.c]
input_sources: []
sync_trees:
  - ../../../shared/gpr_context
control:
  - [XFAIL, "valgrind", "W320-022: GNATCOLL.Projects leaks no longer fixed"]
