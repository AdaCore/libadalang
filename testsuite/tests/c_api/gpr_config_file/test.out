Got an exception (EXCEPTION_PROJECT_ERROR):
  fatal error, cannot load the project tree
empty.cgpr:5:08: error: unrecognized attribute "Unknown_Attribute"
